bus/explorer.go

```diff
@@ -0,0 +1,81 @@
+package bus
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	"go.sia.tech/renterd/api"
+)
+
+type (
+	// An explorer retrieves data about the Sia network from an external source.
+	Explorer interface {
```

Comment:
I'm torn about whether we need this interface here. The type seems simple enough to not require mocking. Only the pin manager should need its own interface to avoid the dependency issue.

Reply:
The `bus` needs it for the `/state` route. Which is a little sad but not really a big deal either?

Reply:
I pushed a commit to demonstrate what I had in mind

Reply:
Thank you, that's very helpful. I thought this was something that we wanted to avoid: https://github.com/SiaFoundation/renterd/pull/1502/commits/6b8d2690e265bae5e5e091f39337eba3a13b49e1#diff-6ef37f8e42ac7f92f5ffee1b700a362cff0a9a70ca71ca7419ac5324e6501499R329 I remember we had a conversation about that couple of weeks ago and I thought we landed on copying interfaces around, no matter how small. Totally fine with that though, thanks!

Reply:
Yeah I'd like to avoid importing large interfaces and instead declare an interface with only the methods we need. 
In this case we bypass that altogether by using the type instead of the interface directly since we are not making use of an interfaces advantages here anyway.